contrib/completion/zsh/_docker

```diff
@@ -78,7 +78,7 @@ __docker_get_containers() {
             s="${${line[${begin[CONTAINER ID]},${end[CONTAINER ID]}]%% ##}[0,12]}"
             s="$s:${(l:15:: :::)${${line[${begin[CREATED]},${end[CREATED]}]/ ago/}%% ##}}"
             s="$s, ${${${line[${begin[IMAGE]},${end[IMAGE]}]}/:/\\:}%% ##}"
-            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Exit* ]]; then
+            if [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Exit* ]] || [[ ${line[${begin[STATUS]},${end[STATUS]}]} = Created* ]] ; then
```

Comment:
I tried this, meanwhile fearing github would explode killing everyone in the neighbourhood, but when I look at https://github.com/docker/docker/pull/31891/files it seems that it worked...